database.yml

```diff
@@ -1,25 +1,27 @@
 mysql:
   dialect: "mysql"
   database: "pop_test"
-  host: {{ envOr "MYSQL_HOST" "127.0.0.1"  }}
-  port: {{ envOr "MYSQL_PORT" "3306"  }}
-  user: {{ envOr "MYSQL_USER"  "root"  }}
-  password: {{ envOr "MYSQL_PASSWORD"  "root#"  }}
+  host: '{{ envOr "MYSQL_HOST" "127.0.0.1"  }}'
+  port: '{{ envOr "MYSQL_PORT" "3306"  }}'
+  user: '{{ envOr "MYSQL_USER"  "root"  }}'
+  password: '{{ envOr "MYSQL_PASSWORD"  "root#"  }}'
   options:
     readTimeout: 5s
+    parseTime: true
+    multiStatements: true
```

Comment:
what is the reason for this change? (single quotes and adding options) I cannot find any description in the PR content. please revert them all if they are not related to the topic of this PR.

Reply:
The pop migrator complained about this in the logs.

Reverted.